setup.py

```diff
@@ -25,7 +25,7 @@
 ]
 
 doc_requires = [
-    "erdantic", # preferable to install with conda, i.e. `conda install erdantic -c conda-forge`
+    "erdantic",  # preferable to install with conda, i.e. `conda install erdantic -c conda-forge`
```

Comment:
Did it fail to install for you with pip? It may have failed for me…I had a messy setup experience and didn’t retry from a clean environment. It’s possible that we should provide dev and user conda environment.yaml files.

Reply:
I didn't try pip based on what their website said.

Reply:
Also, looks like github is having a hard time as well (can't install pygraphviz).